backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/CodeGenerator.kt

```diff
@@ -281,9 +281,13 @@ internal class FunctionGenerationContext(val function: LLVMValueRef,
         LLVMBuildStore(builder, value, ptr)
     }
 
-    fun storeAny(value: LLVMValueRef, ptr: LLVMValueRef) {
+    fun storeRef(value: LLVMValueRef, ptr: LLVMValueRef, onStack: Boolean) {
```

Comment:
As far as I see, most of usages pass boolean literal for `onStack` parameter.
Why not add then `storeHeapRef` and `storeStackRef` to make the code more clear?